docs/installation/reference/import.rst

```diff
@@ -0,0 +1,192 @@
+.. _installation_reference_import:
+
+===========================
+Importing documents in bulk
+===========================
```

Comment:
This comment is optional. 
But what do you think about adding a sequence diagram from GH issue to the documentation?